src/containers/EditSubjectFrontpage/components/SubjectpageAccordionPanels.tsx

```diff
@@ -7,23 +7,31 @@
 import { Accordions, AccordionSection } from '@ndla/accordion';
 import { useTranslation } from 'react-i18next';
 import { FormikErrors } from 'formik';
+import { IArticle } from '@ndla/types-draft-api';
+import { ILearningPathV2 } from '@ndla/types-learningpath-api';
 import SubjectpageAbout from './SubjectpageAbout';
 import SubjectpageMetadata from './SubjectpageMetadata';
 import SubjectpageArticles from './SubjectpageArticles';
-import { FormikProperties } from '../../../interfaces';
 import FormikField from '../../../components/FormikField';
-import { DraftApiType } from '../../../modules/draft/draftApiInterfaces';
-import { Learningpath } from '../../../modules/learningpath/learningpathApiInterfaces';
 import { SubjectPageFormikType } from '../../../util/subjectHelpers';
 
 interface Props {
-  editorsChoices: (DraftApiType | Learningpath)[];
+  editorsChoices: (IArticle | ILearningPathV2)[];
   elementId: string;
   errors: FormikErrors<SubjectPageFormikType>;
 }
 
 const SubjectpageAccordionPanels = ({ editorsChoices, elementId, errors }: Props) => {
   const { t } = useTranslation();
+
+  const SubjectPageArticle = () => (
+    <SubjectpageArticles
+      editorsChoices={editorsChoices}
+      elementId={elementId}
+      fieldName={'editorsChoices'}
+    />
+  );
```

Comment:
Hvorfor endret du dette?

Reply:
For å fjerne prop drilling av `form` og `field`. Vi ender opp i å manuelt definere Props-typer for disse to props og det er det jeg ville unngå ved å erstatte det med innhenting av formiki props via context.